sql/core/src/main/scala/org/apache/spark/sql/execution/subquery.scala

```diff
@@ -171,6 +172,44 @@ case class InSubqueryExec(
   }
 }
 
+/**
+ * The physical node of non-correlated EXISTS subquery.
+ */
+case class ExistsExec(
```

Comment:
Can we name it `ExistsSubquery` or `ExistsSubqueryExec`?

Reply:
> Can we name it `ExistsSubquery` or `ExistsSubqueryExec`?

Sure, update to `ExistsSubqueryExec`